spec/controllers/users_controller_spec.rb

```diff
@@ -55,8 +55,8 @@
       @params[:id] = user.id
     end
 
-    it_should_allow_admin_only do
-      expect(assigns[:user]).to eq(user)
+    context "when create_users is active", feature_setting: { create_users: true } do
```

Comment:
In general that's how I'd do it. In this case, there was a set of existing specs below starting at line 308 that turn the `create_users` feature off and test that the routes don't exist, so I added `#edit` and `#update` there.